src/components/input/inputText/inputText.api.ts

```diff
@@ -0,0 +1,14 @@
+import type { IInputComponentProps } from '../inputContainer';
+
+export interface IInputTextProps extends IInputComponentProps {
+    /**
+     * @property {string} [addon] - Text to be rendered inside the addon. If empty or undefined, no addon will be rendered.
+     */
```

Comment:
We don't need the `@property` block tag, the `{string}` type or the name of the property:
```suggestion
    /**
     * Text rendered beside the input field. If empty or undefined, no addon will be rendered.
     */
```